strapi-app/api/guide-categories/config/routes.json

```diff
@@ -0,0 +1,52 @@
+{
+  "routes": [
+    {
+      "method": "GET",
+      "path": "/guide-categories",
```

Comment:
Are we using these routes - guide-categories - anywhere?

Reply:
Not yet.  This is so that we can add a category to the guides down the road if we'd like to filter  by category.  It's not a mandatory field yet, but because it's a many to many datatype, it needs to be declared like this.  If you'd like I can give a demo in the strapi app.  (I remember finding the pattern a bit odd when I was creating the 'guide' object)